src/client/src/pages/AddPagesPage/PageCard/styles.module.css

```diff
@@ -1,17 +1,3 @@
-.plusIcon {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  border: 1.8px solid var(--vscode-editor-foreground);
-  height: 2em;
-  width: 2em;
-  border-radius: 50%;
-}
-
-.plusIcon path {
```

Comment:
shared for consistency when I saw that it was missing the path fill colour in Services Page